internal/store/job.go

```diff
@@ -152,6 +153,10 @@ var (
 					},
 				}
 			}),
+			AggregateBy: map[string]aggregation.Aggregation{
+				"namespace": aggregation.ByLabels("namespace"),
+				"owner": aggregation.ByNamespaceAndOwner(),
```

Comment:
OK, I guess that "owner" is not there as a label, so some generic code is required to sum by owner, right?